ios/Classes/Plugin/PluginController.swift

```diff
@@ -34,13 +35,14 @@ final class PluginController {
             onDiscovery: papply(weak: self) { context, _, peripheral, advertisementData, rssi in
                 guard let sink = context.scan?.sink
                 else { assert(false); return }
-                
+
                 let serviceData = advertisementData[CBAdvertisementDataServiceDataKey] as? ServiceData ?? [:]
                 let manufacturerData = advertisementData[CBAdvertisementDataManufacturerDataKey] as? Data ?? Data();
+                let localName = advertisementData[CBAdvertisementDataLocalNameKey] as? String ?? peripheral.name ?? String();
```

Comment:
I'm not sure we should still call this "local name", because the fall back is described in the iOS docs as the name registered in a GAP database. Just `name` wouldn't cause this questions.

This is a small detail, but also not difficult to correct. I can do it myself tomorrow.